docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -74,101 +74,230 @@ ml_train_model_construct(&model);
-| `ML_TRAIN_LAYER_TYPE_CONV2D`     |                             |                      | 2D convolution layer                                                  |
-| &#xfeff;                         | filters                     | (integer)            | number of filters                                                     |
-| &#xfeff;                         | kernel_size                 | (integer array)      | comma-separated integers for kernel size,  `height, width` respectively |
-| &#xfeff;                         | stride                      | (integer array)      | comma-separated integers for strides, `height, width` respectively    |
-| &#xfeff;                         | padding                     | (integer array)      | comma-separated integers for padding, `height, width` respectively    |
-| `ML_TRAIN_LAYER_TYPE_POOLING2D`  |                             |                      | 2D pooling layer                                                      |
-| &#xfeff;                         | pooling                     | (categorical)        | type of pooling                                                       |
-| &#xfeff;                         |                             | max                  | max pooling                                                           |
-| &#xfeff;                         |                             | average              | average pooling                                                       |
-| &#xfeff;                         |                             | global_max           | global max pooling                                                    |
-| &#xfeff;                         |                             | global_average       | global average pooling                                                |
-| `ML_TRAIN_LAYER_TYPE_FLATTEN`    |                             |                      | Flatten layer                                                         |
-| `ML_TRAIN_LAYER_TYPE_ACTIVATION` |                             |                      | Standalone, simple activation layer                                   |
-| `ML_TRAIN_LAYER_TYPE_ADDITION`   |                             |                      | Addition layer                                                        |
-| `ML_TRAIN_LAYER_TYPE_CONCAT`     |                             |                      | Concatenation layer                                                   |
-| `ML_TRAIN_LAYER_TYPE_MULTIOUT`   |                             |                      | Multiout layer, repeats output to the number of layers                 |
-
+There are 2 types of layers.
+One type includes commonly trainable weigths and another type does not includes.
+The following are the available properties for each layer type which include commonly trainable weights.
+
+Type | Key | Value | Default value | Description
+---------- | --- | ----- | ----------- | -----------
```

Comment:
Some of the values are in brackets some are not in, any specific reason?
Recommend to follow consistency.

Reply:
discussed in https://github.com/Samsung/tizen-docs/pull/1516#discussion_r716361917